go/install/install.go

```diff
@@ -259,3 +258,15 @@ func IsInUse(mountDir string, log Log) bool {
 	}
 	return false
 }
+
+// TerminateApp will stop the Keybase (UI) app
+func TerminateApp(context Context, log Log) error {
+	appExecName := "Keybase"
+	logf := logger.NewLoggerf(log)
+	log.Info("Stopping Keybase app")
+	appPIDs := process.TerminateAll(process.NewMatcher(appExecName, process.ExecutableEqual, logf), 5*time.Second, logf)
```

Comment:
So this will terminate all applications that contain the string `Keybase`?  Is there any way to make it more granular?  

What if someone writes their own application `KeybaseVideoChat`...this will kill it.  Or during development, this would kill `Keybase` and `KeybaseDevel`.